.github/workflows/lint.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
         uses: golangci/golangci-lint-action@v3.7.0
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
```

Comment:
Is this the case? We seem to be specifying the patch version anyway.

Reply:
You are right. I had not noticed that. I can try and see if 1.54 would work, and we could leave that.

Security wise we should go the other way around, not just lock to the version, but to the sha of the action.

Reply:
seems to work just fine